app/views/layouts/base.html.erb

```diff
@@ -42,7 +42,7 @@
     </div>
     <% end %>
 
-    <h1><%= page_header_title %></h1>
+    <%= image_tag('foremanlogo.png') %>
```

Comment:
While a nice idea, this would make all pages look the same, even in a sub-project, and lose the ability to navigate the project tree.